src/Tools/GetDocumentInsider/src/ReporterExtensions.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using Microsoft.Extensions.Tools.Internal;
+
+namespace Microsoft.Extensions.ApiDescription.Tool
+{
+    internal static class ReporterExtensions
```

Comment:
Why not just write your own type that inherits from ConsoleReporter? Seems sketchy to have a global static to control prefixing the output.

Reply:
There is one but it was specific to `dotnet-watch` until @ryanbrandenburg checked in the OpenAPI global tool (CLI for adding references that use the Microsoft.Extensions.ApiDescription.Client infrastructure). So, the timing didn't work out but we could reconsider in 3.1.

Will file a follow-up issue for this and your `catch` clarity suggestion.